test-proxy/README.md

```diff
@@ -0,0 +1,46 @@
+# CBT Java Test Proxy
+
+The CBT test proxy is intended for running confromance tests for Cloug Bigtable Java Client.
+
+## Set up 
+
+If you have not already done so, [install golang](https://go.dev/doc/install), then clone the go test library:
+
+```
+git clone https://github.com/googleapis/cloud-bigtable-clients-test.git
+```
+
+## Start test proxy
+
+Build the proxy with the latest version of the client
+
+```
+cd java-bigtable/test-proxy
+mvn clean install
+```
+
+Start the proxy on default port 9999
+
+```
+mvn exec:java -Dexec.mainClass=com.google.cloud.bigtable.testproxy.CbtTestProxyMain
+```
+
+Start the proxy on a different port
+
+```
+mvn exec:java -Dexec.mainClass=com.google.cloud.bigtable.testproxy.CbtTestProxyMain -Dport=1
+```
+
+Build and start the proxy with an older version of the client
+
+```
+mvn clean install -Dbigtable.client.version=2.1.0 -Denforcer.skip
```

Comment:
nit - I would replace 2.1.0 with <client_version> so the developer is intentional about which version they pick